python/nano/src/bigdl/nano/tf/keras/amp/bfloat16.py

```diff
@@ -19,12 +19,16 @@
 
 
 def BF16Model(model):
+    model_policys = []
```

Comment:
could we call it `original_model_policies`, seems more understandable. (no strong opinion on this, feel free to ignore)

Reply:
I agree, has updated :D